### ISSUE TITLE
CKPT url not working

### ISSUE THREAD

**Issue by CristianViSa** [iro-cp/FCRN-DepthPrediction#98]
Hi, i am trying to tdownload the tensorflow model but the link is not working.

**Comment by hyi** [comment]
Yes, I am trying to download the tensorflow model from https://campar.in.tum.de/files/rupprecht/depthpred/NYU_FCRN-checkpoint.zip, but the link is not working. I am very interested in trying your model for a project I am working on and hope someone can fix this soon. Thanks

**Comment by SSophies** [comment]
Hi, the link is still not working. Did you find the files somewhere else maybe? Thanks!

**Comment by iladshyan** [comment]
This is still not working

**Comment by wayne931121** [comment]
Hello I can tell you how to find trained model. First see the link in github is http://campar.in.tum.de/files/rupprecht/depthpred/NYU_FCRN-checkpoint.zip

Second Search NYU_FCRN-checkpoint.zip in google, and then you will find some result like https://storage.openvinotoolkit.org/repositories/open_model_zoo/public/2022.1/fcrn-dp-nyu-depth-v2-tf/?sort_by=BIG2SMALL
